spec/spec_helper.rb

```diff
@@ -26,6 +26,8 @@
   c.configure_rspec_metadata!
   c.filter_sensitive_data('%SHIPENGINE_API_KEY%') { ENV['SHIPENGINE_API_KEY'] }
   c.filter_sensitive_data('%SHIPENGINE_CARRIER_ID%') { ENV['SHIPENGINE_CARRIER_ID'] }
+  c.filter_sensitive_data('%SHIPENGINE_LTL_CARRIER_ID%') { ENV.fetch('SHIPENGINE_LTL_CARRIER_ID', 'abc123') }
+  c.filter_sensitive_data('%SHIPENGINE_LTL_CARRIER_SCAC%') { ENV.fetch('SHIPENGINE_LTL_CARRIER_SCAC', 'UPGF') }
```

Comment:
@mamhoff this is the only way I could get the specs to pass without env vars in CircleCI. I have to provide a default value both here and in `ship_engine_ltl_spec.rb` to ensure missing env vars don't cause the spec to fail. Otherwise, the env vars are `nil` and the URLs in the VCR cassettes become invalid.

Reply:
Hm. Do we have `dotenv` in the test environment? We could also add some fake things in the `env.test` file.

Reply:
Good idea. We do. I moved the defaults to `.env.test`